cypress/component/Page.spec.tsx

```diff
@@ -1,13 +1,66 @@
 import React from 'react'
 import type { PageSidebarProps } from '@toptal/picasso'
 import { Container, Menu, Page, Typography } from '@toptal/picasso'
+import type { BreakpointKeys } from '@toptal/picasso-provider/Picasso/config'
+import { PicassoBreakpoints } from '@toptal/picasso-provider/Picasso/config'
 
 const component = 'Page'
 const containerHeight = '30rem'
 
+/**
+ * Produces an array of checkpoints – screen size values, needed for covering
+ * all existing breakpoints in tests (checkpoints are different from breakpoints
+ * by certain offset).
+ *
+ * @returns {number[]} Array of checkpoints
+ */
+const getCheckpoints = () => {
+  const offset = 5
+  const breakpoints = PicassoBreakpoints.breakpoints.values
+  const breakpointKeys = Object.keys(breakpoints)
+
+  return [
+    ...breakpointKeys
+      .filter(breakpointKey => breakpoints[breakpointKey as BreakpointKeys] > 0)
+      .map(
+        breakpointKey => breakpoints[breakpointKey as BreakpointKeys] - offset
+      ),
+    breakpoints[breakpointKeys.slice(-1) as unknown as BreakpointKeys] + offset,
+  ]
```

Comment:
I think I understood this code, but maybe we could make it more clear if we simplified and add some comments to it, wdyt?
```suggestion
  const breakpointValues = Object.values(breakpoints).sort()

  return [
    ...breakpointValues
      .slice(1) // First breakpoint (0 to x) doesn't generate a checkpoint, as it is comprehends anything lower than the second
      .map(breakpoint => breakpoint - offset), // We subtract an offset so the width would be inside the range
    breakpointValues.slice(-1)[0] + offset,
```

Reply:
We could also use named functions instead of line comments, to convey the motivation behind the choices, that are not obvious at first glance

Reply:
@augustobmoura Comment were added, thanks!